src/include/msccl/msccl_struct.h

```diff
@@ -14,7 +14,9 @@
 #include "device.h"
 #include "msccl/msccl_scheduler.h"
 
+#ifndef MSCCL_MAX_NUM_STEPS
```

Comment:
Shall we just remove `MSCCL_MAX_NUM_STEPS` here since it's duplicated macro definition now

Reply:
Agree. Now MSCCL_MAX_NUM_STEPS is removed from msccl_struct.h